### PR TITLE
python3Packages.duecredit: init at 0.8.0

### PR DESCRIPTION
**Pull request by bcdarwin**
python3Packages.citeproc-py: init at 0.5.1
python3Packages.rnc2rng: init at 2.6.4

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [NA] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [NA] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
 `bin/duecredit` with no arguments gives an error due to https://bugs.python.org/issue16308
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bcdarwin** [comment]
@grahamcofborg build python3Packages.duecredit python38Packages.duecredit

File: pkgs/development/python-modules/citeproc-py/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    sha256 = "00aaff50jy4j0nakdzq9258z1gzrac9baarli2ymgspj88jg5968";
+  };
+
+  buildInputs = [ rnc2rng ];
+
+  propagatedBuildInputs = [ lxml ];
+
+  checkInputs = [ nose git ];
+  checkPhase = "nosetests tests";
+  doCheck = false;  # seems to want a Git repository, but fetchgit with leaveDotGit also fails
```
Comment (jonringer) [inline pkgs/development/python-modules/citeproc-py/default.nix:25]: If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors.

Comment (bcdarwin) [reply pkgs/development/python-modules/citeproc-py/default.nix:25]: Done, thanks.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 89971 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>8 packages built:</summary><br>- python27Packages.citeproc-py<br>- python27Packages.rnc2rng<br>- python37Packages.citeproc-py<br>- python37Packages.duecredit<br>- python37Packages.rnc2rng<br>- python38Packages.citeproc-py<br>- python38Packages.duecredit<br>- python38Packages.rnc2rng</details>

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 89971 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>8 packages built:</summary><br>- python27Packages.citeproc-py<br>- python27Packages.rnc2rng<br>- python37Packages.citeproc-py<br>- python37Packages.duecredit<br>- python37Packages.rnc2rng<br>- python38Packages.citeproc-py<br>- python38Packages.duecredit<br>- python38Packages.rnc2rng</details>

**Pull request merged**